src/oc/nrnran123.h

```diff
@@ -40,19 +40,26 @@ extern uint32_t nrnran123_get_globalindex();
 /* minimal data stream */
 extern nrnran123_State* nrnran123_newstream(uint32_t id1, uint32_t id2);
 extern nrnran123_State* nrnran123_newstream3(uint32_t id1, uint32_t id2, uint32_t id3);
+extern nrnran123_State* nrnran123_newstream0();  // increment id3 for each instance.
 extern void nrnran123_deletestream(nrnran123_State*);
 extern void nrnran123_getseq(nrnran123_State*, uint32_t* seq, char* which);
 extern void nrnran123_setseq(nrnran123_State*, uint32_t seq, char which);
+extern void nrnran123_setseq1(nrnran123_State*, double seq4which);  // seq*4+which);
 extern void nrnran123_getids(nrnran123_State*, uint32_t* id1, uint32_t* id2);
 extern void nrnran123_getids3(nrnran123_State*, uint32_t* id1, uint32_t* id2, uint32_t* id3);
+extern void nrnran123_setids(nrnran123_State*, uint32_t id1, uint32_t id2, uint32_t id3);
 
-extern double nrnran123_negexp(nrnran123_State*);  /* mean 1.0 */
+extern double nrnran123_negexp(nrnran123_State*); /* mean 1.0 */
+extern double nrnran123_negexp(nrnran123_State*, double);
```

Comment:
@pramodk I guess you added the distribution arg version of negexp, normal, and uniform. As they are not needed for the "generator only"  simplified PR, should we revert them to nonexistence. Or should we keep and document for use in mod files eg.
```
value = random_negexp(ranvar) : mean 1
value = random_negexp(ranvar, mean)
value = random_normal(ranvar) : mean 1, std 1
value = random_normal(ranvar, mean, std)
value = random_uniform(ranvar) : open interval (0-1)
value =random_uniform(ranvar, min, max) : open interval (min, max)
```